jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -690,6 +690,8 @@ int jbmc_parse_optionst::get_goto_program(
   std::unique_ptr<abstract_goto_modelt> &goto_model_ptr,
   const optionst &options)
 {
+  if(options.is_set("context-include") || options.is_set("context-exclude"))
```

Comment:
It's ugly to repeat that here, but probably can't be fixed without cleaning up the language interface.